### PR TITLE
Nikhil's change for updating multiplier to shifter

### PR DESCRIPTION
**Pull request by norabarlow**
Changes multiplier used to calculated FP complex ops to a shifter (better for area, power...)

Would be nice to get this in as soon as possible for RTL freeze.

### CODE REVIEW

File: lassen/alu.py [hunk 1]
```diff
@@ -158,7 +158,8 @@ def __call__(self, alu: ALU_t, signed_: Signed_t, a: Data, b: Data, d:Bit) -> (D
                 if (abs_exp[7] == Bit(1)):
                     scale = SInt[16](7)
                 normmant_mul_left = SInt[16](abs_exp)
-                normmant_mul_right = SInt[16](1) << (SInt[16](7)-scale)
+                #normmant_mul_right = SInt[16](1) << (SInt[16](7)-scale)
```
Comment (rdaly525) [inline lassen/alu.py:161]: remove

File: lassen/alu.py [hunk 2]
```diff
@@ -206,13 +207,16 @@ def __call__(self, alu: ALU_t, signed_: Signed_t, a: Data, b: Data, d:Bit) -> (D
                 if (abs_input[15] == Bit(1)):
                     scale = SInt[16](15)
                 normmant_mul_left = SInt[16](abs_input)
-                normmant_mul_right = (SInt[16](1) << (SInt[16](15)-scale))
+                #normmant_mul_right = (SInt[16](1) << (SInt[16](15)-scale))
```
Comment (rdaly525) [inline lassen/alu.py:210]: remove

File: lassen/alu.py [hunk 3]
```diff
@@ -206,13 +207,16 @@ def __call__(self, alu: ALU_t, signed_: Signed_t, a: Data, b: Data, d:Bit) -> (D
...
+                normmant_mul_right = (SInt[16](15)-scale)
                 normmant_mask = SInt[16](0x7f00)
 
             #if (alu == ALU_t.FCnvInt2F) | (alu == ALU_t.FCnvExp2F):
             if (scale >= 0):
+                print("scale ")
```
Comment (rdaly525) [inline lassen/alu.py:216]: Please no extraneous print statements. Also this is likely the cause of the travis failure

**Comment by rsetaluri** [comment]
Going to work from #169. Closing

**Pull request closed without merging**